core-web/libs/dotcms-models/src/lib/dot-contentlet.model.ts

```diff
@@ -18,7 +21,7 @@ export interface DotCMSContentlet {
     inode: string;
     image?: string;
     languageId: number;
-    language?: string;
+    language?: string | DotLanguage;
```

Comment:
the endpoint now return DotLanguage ?

Reply:
Yes, the language is not always included, but in some cases, it is. For example, when I use this endpoint `/api/v1/content/<id>?depth=2` to retrieve the content relationships, the language is included.”

<img width="558" alt="Screenshot 2025-01-09 at 11 59 59 AM" src="https://github.com/user-attachments/assets/d3cc5727-fcc7-49f5-bea3-d81bc506d6b1" />